README.md

```diff
@@ -7,6 +7,15 @@
 
 # Zurich Instruments Labber Drivers
 
+> **END OF LIFE**
+>
+> LabOne 24.10 will be the last LabOne version officially supported by the
+> Zurich Instruments Labber drivers. After LabOne 24.10 we do no longer
+> test and maintain the drivers.
+> 
+> For existing users we can provide support until end of 2025.
```

Comment:
Slightly changing the tone: "Zurich Instruments continues to support the existing users of Labber drivers until the end of 2025."